velox/exec/Task.h

```diff
@@ -747,6 +747,9 @@ class Task : public std::enable_shared_from_this<Task> {
     return cancellationSource_.getToken();
   }
 
+  /// Returns the driver by driverIdx in drivers_.
+  std::shared_ptr<Driver> getDriver(uint32_t driverIdx) const;
+
```

Comment:
I guess it's a big change to expose the drivers. There's `testingVisitDrivers` can be used if it's just to work around something.

Reply:
Thanks for this information, I will try.